### PR TITLE
August Release - part 1

### PR DESCRIPTION
**Pull request by sukhada**
This release includes the following tickets:

1. https://github.com/350org/action-tools/issues/264
2. https://github.com/350org/ak_intl_v3/issues/429
3. https://github.com/350org/ak_intl_v3/issues/428
4. https://github.com/350org/ak_intl_v3/issues/432
5. https://github.com/350org/ak_intl_v3/issues/431
6. https://github.com/350org/action-tools/issues/257

### CODE REVIEW

**Review by cjmabry — COMMENTED** [review]
Quick question otherwise looks good!

**Comment by sukhada** [comment]
It's in master - I checked already!

File: afteraction-share.html [hunk 1]
```diff
@@ -201,16 +201,17 @@ <h2>{{ page.custom_fields.share_slide_title }}</h2>
...
-	{% if page.custom_fields.share_whatsapp_message %}
+  {% if page.custom_fields.share_whatsapp_message %}
 		<script>
-		var whatsappMsg = '{{ page.custom_fields.share_whatsapp_message }} ';
+		var whatsappMsg = '{{ page.custom_fields.share_whatsapp_message|escapejs }} ';
 		</script>
 	{% else %}
 		<script>
-		var whatsappMsg = '{{ page.followup.share_title_value }} ';
+		var whatsappMsg = '{{ page.followup.share_title_value|escapejs }} ';
```
[the remark this answers is not in the document]
Comment (sukhada) [reply afteraction-share.html:210]: @cjmabry it's in master already!

File: wrapper.html [hunk 2]
```diff
@@ -52,6 +52,22 @@
 		})(window, document, 'script', 'dataLayer', 'GTM-MNNGDHL');
 	</script>
 	<!-- End Google Tag Manager -->
+	<!-- Start VWO Async SmartCode -->
```
Comment (cjmabry) [inline wrapper.html:55]: @sukhada this is in master as well?

Comment (sukhada) [reply wrapper.html:55]: yep: https://github.com/350org/ak_intl_v3/commit/16a875a13aa1cd81257b7947a206451e08b6b6ca

**Review by cjmabry — APPROVED** [review]

**Pull request merged**